### ISSUE TITLE
arangorestore just fail reloading data for this specific export

### ISSUE THREAD

**Issue by solisoft** [arangodb/arangodb#4885]
## my environment running ArangoDB
I'm using the latest ArangoDB of the respective release series:
- [X] 3.3

Mode:
- [X] Single-Server

Storage-Engine:
- [X] rocksdb

On this operating system:
- [x] Linux 
  - [X] docker - official docker library
- [X] MacOS, version:

The issue can be reproduced using this dataset:
[cat.tar.gz](https://github.com/arangodb/arangodb/files/1822475/cat.tar.gz)

**Comment by vinaypyati** [comment]
Hi @solisoft,

Thank you for reporting the issue. I tried restoring the given dataset with the version 3.3.4 and it turns out that the restoration works fine both on Linux and Windows. 
Can you please share the error log, commands and the version that you are using for restoration? 

Thanks.

**Comment by solisoft** [comment]
The command I ran on my mac (and my linux) was 
`arangorestore --input-directory thebrane --server.database thebrane2 --overwrite true`

The log are : 
```
Please specify a password:
Database name in source dump is 'thebrane'
Server version: 3.3.4
# Connected to ArangoDB 'http+tcp://127.0.0.1:8529'
# Re-creating edge collection 'categories_path'...
# Loading data into edge collection 'categories_path'...
2018-03-17T15:34:23Z [24592] ERROR internal error: got invalid response from server: Error reading from: 'http+tcp://127.0.0.1:8529' 'timeout during read'
```

**Comment by vinaypyati** [comment]
Hi @solisoft,

Thank you for the commands and error log. After trying restoration with ```--overwrite``` the error is now reproducible. The restoration takes a long time( ~40mins ) for a dump of 33MB. We will further analyze this issue with the dataset that you have provided. 

Thanks.

**Comment by marjones** [comment]
Is there any Update on this issue?

**Comment by graetzer** [comment]
Sorry I will look at this today

**Comment by graetzer** [comment]
Hey, I am really sorry about taking so long for this.
It seems your data is generating an error while importing: `unique constraint violated - in index 15785406 of type rocksdb-skiplist over [\"_from\",\"_to\"]; conflicting key: 84909169"`

Not sure why it is not shown to you directly, I will investigate

**Comment by graetzer** [comment]
For the meantime try specifying `--batch-size  65536` or something like that, this should help a little

**Comment by vinaypyati** [comment]
Hi @solisoft ,

Did you have a chance to restore the dataset using the workaround suggested by graetzer?

> try specifying `--batch-size 65536` or something like that, this should help a little


Thanks!

**Comment by solisoft** [comment]
Sorry, didn't get time yet :/

**Comment by OmarAyo** [comment]
Hi @solisof 

Just sending a quick followup - how everything is going? 
Does the suggested workaround works for you?

Best,

**Comment by OmarAyo** [comment]
Hi @solisoft 

Could you please share with us a quick feedback on this ticket 

Thanks,

**Comment by solisoft** [comment]
I'm not sure the test will be relevant ... I made some database upgrade ...

**Comment by maxkernbach** [comment]
Hi @solisoft,

are we set to close this issue, has the database upgrade fixed the arangorestore problem?

**Comment by solisoft** [comment]
Issue is fixed (tested on 3.3.1O) Thanks
